orange_ecom_starter.theme

```diff
@@ -548,3 +556,10 @@ function orange_ecom_starter_preprocess_swiftmailer(&$variables) {
 function orange_ecom_starter_preprocess_commerce_cart_block(&$variables) {
   $variables['attributes']['class'][] = 'cart--cart-block';
 }
+
+/**
+ * Implements hook_commerce_checkout_pane_info_alter(&$checkout_panes).
+ */
+function orange_ecom_starter_commerce_checkout_pane_info_alter(&$checkout_panes) {
+  if($checkout_panes['shipping_information']) { return false; }
+}
```

Comment:
The file doesn't end with a newline. That's required by the code standard. This is easier if you set PhpStorm to always add it automatically (in Settings > Editor > General, select "Ensure line feed at file end on Save").

Reply:
Fixed